runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/fn/data/BeamFnDataGrpcServiceTest.java

```diff
@@ -284,8 +284,8 @@ public void testMultipleClientsSendMessagesAreDirectedToProperConsumers() throws
     return BeamFnApi.Elements.newBuilder()
         .addData(
             BeamFnApi.Elements.Data.newBuilder()
-                .setInstructionReference(id)
-                .setPtransformId(PTRANSFORM_ID)
+                .setInstructionId(id)
+                .setTransformId(PTRANSFORM_ID)
```

Comment:
TRANSFORM_ID ?

Reply:
I was going to leave changes like this for another PR because it will be easier to do as a follow-up since it won't touch all the languages and require running as many tests. Sound good?

Reply:
SGTM.